app/javascript/packs/admin_evaluations/new_evaluation.jsx

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
```

Comment:
in general, this component way to big and I think you spent a lot of time on it because of its size. It is better to split it to smaller ones.